lib/zendesk_apps_support/validations/secrets.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ZendeskAppsSupport
+  module Validations
+    module Secrets
+      SECRET_KEYWORDS = %w[
+        pass password secret secretToken secret_token auth_key
```

Comment:
Just something worth noting, the matcher for these keywords just checks if they're present in the file.
While testing this gave a false positive as `passed` was present in a file which matched against `pass`. Just a small thing worth noting which could annoy devs but definitely not a showstopper imo.